addon/maptools/rectangletool.js

```diff
@@ -3,31 +3,35 @@ import MapTool from 'ember-flexberry-gis/maptools/maptool';
 export default MapTool.extend({
 
   _layer: null,
+  _measure: null,
 
-  rectangleStarted({ layer }) {
```

Comment:
What functionality supports this function?
Should I support this finctionality in other measurements scripts:
circletool.js, markertool.ls, polyline.js, polygon.js?
Or i should remain primary code in rectangletool.js and support measurement function in other scritps: rectangeMeasureTool.js, markerMeasureTool.js, circleMeasureTool.js etc...?